### DISCUSSION TITLE
IWAS

### DISCUSSION THREAD

**Discussion by giscus** [aduros/wasm4#778] [Games]
# IWAS

Build retro games using WebAssembly for a fantasy console

https://wasm4.org/play/iwas

**Comment by potentialnull** [comment]
Really cool! I made a neat 1 page loop. Is there a way to loop a specific page?

**Reply by nickjrotundo** [reply]
If you click the play button that's like |⏵︎| and then also the loop button 🔁, you can loop a specific page
